monitor/dbt_project/macros/get_new_model_alerts.sql

```diff
@@ -1,8 +1,19 @@
-{% macro get_new_model_alerts(days_back, results_sample_limit = 5) %}
+{% macro get_new_model_alerts(days_back) %}
     -- depends_on: {{ ref('alerts_models') }}
     {% set select_new_alerts_query %}
-        SELECT * FROM {{ ref('alerts_models') }}
-        WHERE alert_sent = FALSE and detected_at >= {{ get_alerts_time_limit(days_back) }}
+        WITH alerts AS (
+            SELECT * FROM {{ ref('alerts_models') }}
+            WHERE alert_sent = FALSE and detected_at >= {{ get_alerts_time_limit(days_back) }}
+        ),
+
+        models AS (
+            SELECT * FROM {{ ref('elementary', 'dbt_models') }}
+        )
+
+        SELECT
+            alerts.*,
+            models.meta as meta
```

Comment:
I think this is a bug because in the normalize_alert you extract the channel only from model_meta and not from meta